src/tests/D3MHub.t.sol

```diff
@@ -1611,25 +1175,25 @@ contract D3MHubTest is DssTest {
         vat.file(ilk, "line", 60 * RAD);
 
         // Simulate a loss event by removing the share tokens
```

Comment:
Hmm for sure this comment is misleading now. But another point to make me wonder if we can fallback to something more similar it was before.

Reply:
After discussions, let's just update to this:
```suggestion
        // Simulate a loss event by removing the tokens
```